src/sentry/static/sentry/app/views/eventsV2/querycard.tsx

```diff
@@ -39,23 +40,24 @@ class QueryCard extends React.PureComponent<Props> {
       queryDetail,
       renderContextMenu,
       renderGraph,
+      createdBy,
     } = this.props;
 
     return (
       <Link data-test-id={`card-${title}`} onClick={this.handleClick} to={this.props.to}>
         <StyledQueryCard interactive>
           <QueryCardHeader>
-            <CardHeading>
-              {title}
-              {starred && (
-                <StyledIconBookmark
-                  color={theme.yellow}
-                  data-test-id="is-saved-query"
-                  solid
-                />
-              )}
-            </CardHeading>
-            <StyledQueryDetail>{queryDetail}</StyledQueryDetail>
+            <QueryCardContent>
+              <QueryTitle>{title}</QueryTitle>
+              <QueryDetail>{queryDetail}</QueryDetail>
+            </QueryCardContent>
+            {starred ? (
+              <StyledUserAvatar user={createdBy} />
+            ) : (
+              <Avatar>
```

Comment:
There is another `SystemAvatar` in app/components/activity/item/avatar.tsx. Would you want to consolidate that with this?

Reply:
What's the difference between SystemAvatar and UserAvatar in this sense?